### PR TITLE
Add option to cli to list only forks or sources

### PR DESCRIPTION
**Pull request by forivall**
As in, `github-repositories --forks no forivall|cut -d' ' -f2|xargs -n1 git clone`

### CODE REVIEW

**Comment by sindresorhus** [comment]
Good idea, but I think it would be better to go with two flags, `--forks` and `--sources`.

**Comment by forivall** [comment]
Yeah, I thought of that halfway through writing it :skull:. I'll see what @kevva's opinion is.

File: cli.js [hunk 1]
```diff
@@ -23,6 +24,17 @@ if (!cli.input[0]) {
 	process.exit(1);
 }
 
+var listForks = null;
+if (cli.flags.forks) {
+	switch (cli.flags.forks) {
+		case 'y': case 'yes': case 'true': case 't': listForks = true; break;
+		case 'n': case 'no': case 'false': case 'f': listForks = false; break;
```
Comment (eush77) [inline cli.js:31]: Why didn't you go with [boolean options](https://github.com/substack/minimist/blob/4cf45a26b9af5f4ddab63107f4007485e576cfd3/test%2Fbool.js)? You wouldn't have to parse all these cases manually, and you would also get `--no-forks` for free.

**Comment by sindresorhus** [comment]
// @kevva

**Comment by kevva** [comment]
Thanks!

**Pull request closed without merging**